ciao-controller/internal/quotas/quotas.go

```diff
@@ -313,6 +322,11 @@ func (qs *Quotas) Init() {
 				dumpData := data.(*dumpOp)
 				dumpData.ch <- dump(tenantDetails, dumpData)
 				close(dumpData.ch)
+
```

Comment:
Note there's a little trick you can do with a type switch to avoid the explicit cast,

```
switch op := data.(type) {
   case *deleteTenantOp:
      // The type of op in this case statement is *deleteTenantOp
}
```